luacheck/luacheck.lua

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env tarantool
+package.path = package.path .. ";/usr/share/tarantool/luacheck/?/init.lua;"
+package.path = package.path .. ";/usr/share/tarantool/luacheck/?.lua;"
```

Comment:
It will not with rocks.

Reply:
Can you explain please?

Reply:
will not →will not work

I mean, install path for rocks is other, it containts `.rocks`. Moreover, install path for modules can be, say, /usr/local/…

Reply:
Removed absolute paths.